tests/cephfs/cephfs_bugs/test_validate_mclientcaps_revoke_ack.py

```diff
@@ -0,0 +1,231 @@
+        )
+        out, rc = clients[0].exec_command(
+            sudo=True,
+            cmd="ceph config get mds debug_mds",
+        )
+        log.info(out)
+        if "20" not in out.strip():
+            log.error("Unable to set debug_mds")
+            return 1
+
+        subvolumegroup = {
+            "vol_name": default_fs,
+            "group_name": subvolume_group_name,
+        }
+        fs_util_v1.create_subvolumegroup(clients[0], **subvolumegroup)
+        subvolume_list = [
+            {
+                "vol_name": default_fs,
+                "subvol_name": f"{subvolume_name}_1",
+                "group_name": subvolume_group_name,
+            },
+            {
+                "vol_name": default_fs,
+                "subvol_name": f"{subvolume_name}_2",
+                "group_name": subvolume_group_name,
+            },
+        ]
+        for subvolume in subvolume_list:
+            fs_util_v1.create_subvolume(clients[0], **subvolume)
+
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        log.info("Mount 1 subvolume on kernel and 1 subvolume on Fuse → Client1")
+
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ips = fs_util_v1.get_mon_node_ips()
+        subvol_path_kernel, rc = clients[0].exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {default_fs} {subvolume_name}_1 {subvolume_group_name}",
+        )
+        fs_util_v1.kernel_mount(
+            [clients[0]],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+            sub_dir=f"{subvol_path_kernel.strip()}",
+            extra_params=f",fs={default_fs}",
+        )
+
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        subvol_path_fuse, rc = clients[0].exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {default_fs} {subvolume_name}_2 {subvolume_group_name}",
+        )
+        fs_util_v1.fuse_mount(
+            [clients[0]],
+            fuse_mounting_dir_1,
+            extra_params=f" -r {subvol_path_fuse.strip()} --client_fs {default_fs}",
+        )
+
+        file_paths = [
+            f"/mnt/cephfs_kernel{mounting_dir}_1/large_file_kernel_1.txt",
+            f"/mnt/cephfs_fuse{mounting_dir}_1/large_file_fuse_1.txt",
+        ]
+
+        create_large_file_script = "create_large_file.py"
+        clients[0].upload_file(
+            sudo=True,
+            src="tests/cephfs/cephfs_bugs/create_large_file.py",
+            dst=f"/root/{create_large_file_script}",
+        )
+        clients[0].exec_command(
+            sudo=True,
+            cmd=f"python /root/{create_large_file_script} {' '.join(file_paths)}",
+            timeout=7200,
+        )
+
+        log.info(
+            "Check for the Ceph Health to see if there are any deadlock bw unlink and rename."
```

Comment:
how are we ensuring unlink and rename ops are triggered?

Reply:
Sorry, the sentence is wrong. We are just checking the Health is OK, if anything other than OK , we are raising exception in "def get_ceph_health_status"